__tests__/app.test.js

```diff
@@ -145,4 +145,12 @@ describe('6', () => {
                 }))
         })
     });
+});
+
+describe('12. DELETE /api/comments/:comment_id', () => {
+    test('returns 204', () => {
+        return request(app)
+        .delete("/api/comments/19")
+        .expect(204)
+    });
```

Comment:
As with the other endpoints you should also test for:
- [ ] Status 404, non existant ID, e.g 999
- [ ] Status 400, invalid ID, e.g "not-an-id"